### PR TITLE
Pza 18 search bar refinement

### PR DESCRIPTION
**Pull request by sarojghalan**
PZA-18 search bar refinement with material ui Drawer component , where drawer is made dynamic you can pass component as children .

### CODE REVIEW

File: src/components/SearchBar/SearchBar.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { Container } from 'react-bootstrap';
+import '../../scss/main.scss';
```
Comment (dipeshkumarsah98) [inline src/components/SearchBar/SearchBar.tsx:2]: load css the last
- package.
- components
- styles

File: src/components/searchBar/SearchBar.tsx [hunk 2]
```diff
@@ -1,51 +1,18 @@
...
-              <SearchRoundedIcon fontSize="large" />
-            </IconButton>
-          </div>
-        </div>
-      </div>
-    </div>
-  );
-}
+import { Container } from 'react-bootstrap';
+import '../../scss/main.scss';
```
Comment (dipeshkumarsah98) [inline src/components/searchBar/SearchBar.tsx:2]: Import custom style file in the last

File: src/Layouts/nav/Navbar.tsx [hunk 3]
```diff
@@ -1,67 +1,55 @@
-import { useState } from 'react';
+import '../../scss/main.scss';
```
Comment (dipeshkumarsah98) [inline src/Layouts/nav/Navbar.tsx:1]: import style file at the last

File: src/Layouts/nav/Navbar.tsx [hunk 4]
```diff
@@ -1,67 +1,55 @@
...
-import Rightdrawers from 'components/drawers/Rightdrawers';
-import SearchBar from 'components/searchBar/SearchBar';
-
-interface NavBarProps {
-  backgroundState: (state: boolean) => void;
-}
+import { logo } from '../../assets/Images/index';
+import Rightdrawers from '../../components/drawers/Rightdrawers';
+import DrawerProvider from '../../utils/Drawer/DrawerProvider';
+import SearchBar from '../../components/SearchBar/SearchBar';
```
Comment (subhashishshah) [inline src/Layouts/nav/Navbar.tsx:9]: Again why are we using relative import? Please use absolute import

File: src/components/SearchBar/SearchBar.tsx (hunk 1 above)
Comment (subhashishshah) [reply src/components/SearchBar/SearchBar.tsx:2]: and also use absolute import

**Review by subhashishshah — CHANGES_REQUESTED** [review]

File: src/Layouts/nav/Navbar.tsx [hunk 5]
```diff
@@ -1,67 +1,53 @@
...
+import { SearchIcon, LocalMallIcon } from 'assets/icons';
 import { NavLink, useNavigate } from 'react-router-dom';
-import { logo } from 'assets/Images';
-import Rightdrawers from 'components/drawers/Rightdrawers';
 import SearchBar from 'components/searchBar/SearchBar';
-
-interface NavBarProps {
-  backgroundState: (state: boolean) => void;
-}
+import { logo } from '../../assets/Images/index';
```
Comment (subhashishshah) [inline src/Layouts/nav/Navbar.tsx:5]: Absolute import recommended

File: src/components/index.ts [hunk 6]
```diff
@@ -18,7 +18,6 @@ import RegisterForm from './form/RegisterForm';
 import ShippingForm from './form/ShippingForm';
 
 import ModalComponent from './modal/ModalComponent';
-import SearchBar from './searchBar/SearchBar';
 import Title from './Title/Title';
```
Comment (subhashishshah) [inline src/components/index.ts:21]: abosulte import recommended

**Pull request merged**